includes/core-changes/globalization/5.0/icu-globalization-api.md

```diff
@@ -19,7 +19,7 @@ This change was introduced for two reasons:
 
 #### Recommended action
 
-No action is required on the part of the developer. However, if you wish to continue using NLS globalization APIs, you can set a [run-time switch](../../../../docs/core/run-time-config/globalization.md#nls) to revert to that behavior.
+No action is required on the part of the developer. However, if you wish to continue using NLS globalization APIs, you can set a [run-time switch](../../../../docs/core/run-time-config/globalization.md#nls) to revert to that behavior. For more information about the available switches, see the [.NET globalization and ICU](/dotnet/standard/globalization-localization/globalization-icu) article.
```

Comment:
Isn't it better to use a relative link so it's validated at build time?

Reply:
I didn't know that. Happy to create a new PR if you want this.